### DISCUSSION TITLE
macbook 编辑项下面没有“使用模板插入多个条目”

### DISCUSSION THREAD

**Discussion by RyanZong2233** [windingwind/zotero-better-notes#866] [Q&A] [answered]
我用的是macbook，安装插件后，编辑项下面没有“使用模板插入多条笔记”“使用模板插入多个条目”“笔记引用格式”“降低标题层级”“提升标题层级”几个选项，这个是要自己写模板导入吗

**Comment by windingwind** [comment, marked as answer]
部分功能位置改变/移除，请看菜单栏或笔记编辑器内的齿轮菜单。

**Comment by RyanZong2233** [comment]
菜单栏和笔记编辑器内的齿轮菜单都看过了都没有，可能是被移除了。。可惜，这几个功能还蛮好用的

**Reply by windingwind** [reply]
使用模板插入多条笔记”“使用模板插入多个条目” = 插入笔记模板

**Reply by RyanZong2233** [reply]
就是说笔记模板里一开始是空的什么都没有，现在要自己去插入了，可是还不太会写只能去discussion里找别人写好的模板hhh
顺便笔记引用格式也没了，这个好像在discussion里也没找到template

**Reply by windingwind** [reply]
笔记模板本来也需要自己按需导入的。
笔记引用格式由zotero内置的引文功能替代

**Reply by LoveBreeze007** [reply]
你好，请问你解决了么？我也是不知道怎么配置。
